blocks/library/embed/index.js

```diff
@@ -19,17 +19,21 @@ registerBlock( 'core/embed', {
 		caption: children( 'figcaption' )
 	},
 
-	edit( { attributes, isSelected, setAttributes } ) {
+	edit( { attributes, setAttributes, focus, setFocus } ) {
 		const { url, title, caption } = attributes;
 
 		return (
 			<figure>
-				<iframe src={ url } title={ title } />
-				{ caption || isSelected ? (
+				<div className="iframe-overlay">
+					<iframe src={ url } title={ title } />
```

Comment:
Instead of a setting, they could just add a wrapper to place it where they want. Maybe this could be a separate React component like `<IframeOverlay>`.